src/hotel-search/client.js

```diff
@@ -125,7 +127,13 @@ HotelSearchClient.prototype = {
       rateAmenityIds: this.rateAmenityIds,
       offset: this.lastRatesCount,
     };
+
+    if (selectedHotelIds !== undefined && Array.isArray(selectedHotelIds)) {
```

Comment:
because `selectedHotelIds` comes from users like rateAmenityIds, so just need to add `selectedHotelIds: this. selectedHotelIds` in side searchParams

Reply:
Also need to add selectedHotelIds at here https://github.com/wego/wego-sdk-js/pull/39/files#diff-125b1a097ba8f387f44da80667623729R16